app/src/androidTest/java/ch/epfl/sdp/LoginTest.java

```diff
@@ -67,12 +74,12 @@ public void writingPassword_Works() {
         onView(withId(R.id.passwordlog)).perform(typeText(password)).check(matches(withText(password)));
     }
 
-    @Test
+    /*@Test
```

Comment:
Instead of commenting, it's maybe better to simply delete the test

Reply:
that's right thanks!